modules/soe_paragraph_stories/config/install/paragraphs.paragraphs_type.soe_stories.yml

```diff
@@ -0,0 +1,10 @@
+uuid: 6e81ffa8-9797-4da8-851f-00c6a4827502
+langcode: en
+status: true
+dependencies: {  }
+id: soe_stories
```

Comment:
This should be `stanford_stories` to follow our naming conventions.

Reply:
fixed.